09-features.md

```diff
@@ -35,6 +35,8 @@ The Context column decodes as follows:
 | 8/9   | `var_onion_optin`                | Requires/supports variable-length routing onion payloads  | IN       | [Routing Onion Specification][bolt04] |
 | 10/11 | `gossip_queries_ex`              | Gossip queries can include additional information         | IN       | [BOLT #7][bolt07-query]               |
 | 12/13| `option_static_remotekey`         | Static key for remote output                              | IN       | [BOLT #3](03-transactions.md)         |
+| 14/15 | `payment_secret`                 | Node supports `payment_secret` field                      | IN9      | [Routing Onion Specification][bolt04]   |
+| 16/17 | `basic_mpp`                      | Node can receive basic multi-part payments                | IN9      | [BOLT #3][bolt04-mpp]                 |
```

Comment:
what is the rationale for setting this in these in the init message? so that they might be used for informing channel funding?

Reply:
Yeah; since node_announcements are not 100% reliable, this lets you tell about the direct neighbors at least.

Reply:
sounds good, it can't hurt to do so